garak/harnesses/base.py

```diff
@@ -94,6 +94,11 @@ def run(self, model, probes, detectors, evaluator, announce_probe=True) -> None:
             logging.debug("harness: probe start for %s", probe.probename)
             if not probe:
                 continue
+            # TODO: refactor this to allow `compatible` probes instead of direct match
+            if probe.modality["in"] != model.modality["in"]:
+                logging.warning("probe skipped due to modality mismatch: %s", probe)
```

Comment:
add context to the message

```suggestion
                logging.warning("probe skipped due to modality mismatch: %s - model expects %s", (probe, model.modality["in"]))
```